Geometry/RPCGeometryBuilder/python/rpcGeometry_cfi.py

```diff
@@ -1,10 +1,6 @@
-import FWCore.ParameterSet.Config as cms
-
-#
-# This cfi should be included to build the RPC geometry model.
-#
-RPCGeometryESModule = cms.ESProducer("RPCGeometryESModule",
-    useDDD = cms.untracked.bool(True)
-)
+from Geometry.RPCGeometryBuilder.RPCGeometryESModule_cfi import RPCGeometryESModule as RPCGeometryESModuleDefault
```

Comment:
as @makortel commented, `_RPCGeometryESModule` is better than `RPCGeometryESModuleDefault`

Reply:
Now the two ESProducers producing same products lead to failures.